sensor/extrinsic_calibration_manager/launch/aip_xx1/ground_plane_sensors.launch.xml

```diff
@@ -18,17 +21,48 @@
   <node pkg="extrinsic_calibration_manager" exec="extrinsic_calibration_manager" name="extrinsic_calibration_manager" output="screen">
     <param name="parent_frame" value="$(var parent_frame)"/>
     <param name="child_frames" value="
-    [velodyne_rear_base_link]"/>
+    [sensor_kit_base_link]"/>
   </node>
 
   <!-- velodyne_rear_base_link: extrinsic_ground_plane_calibrator -->
```

Comment:
Is this now `sensor_kit_base_link` ?

Reply:
Sorry, what do you mean?
The chain is:
base -> sensor_kit -> lidar_base -> lidar

In this case, both sensor_kit -> lidar_base and lidar_base -> lidar are hardcoded, so the output of base-lidar calibration is base->sensor_kit